fbw-a32nx/src/systems/instruments/src/NDv2/shared/WindIndicator.tsx

```diff
@@ -23,25 +24,25 @@ export class WindIndicator extends DisplayComponent<{ bus: EventBus }> {
     onAfterRender(node: VNode) {
         super.onAfterRender(node);
 
-        const sub = this.props.bus.getSubscriber<AdirsSimVars>();
+        const sub = this.props.bus.getSubscriber<AdirsSimVars & DmcLogicEvents>();
 
-        sub.on('windDirection').whenChanged().atFrequency(2).handle((value) => {
+        sub.on('windDirection').atFrequency(2).handle((value) => {
             this.windDirectionWord.setWord(value);
 
             this.handleWindDirection();
             this.handleWindSpeed();
             this.handleWindArrow();
         });
 
-        sub.on('windSpeed').whenChanged().atFrequency(2).handle((value) => {
+        sub.on('windSpeed').atFrequency(2).handle((value) => {
             this.windSpeedWord.setWord(value);
 
             this.handleWindDirection();
             this.handleWindSpeed();
             this.handleWindArrow();
         });
 
-        sub.on('trueHeading').whenChanged().atFrequency(2).handle((value) => {
+        sub.on('heading').atFrequency(2).handle((value) => {
```

Comment:
This is not correct. Some other parts of the wind code here don't look right either. https://github.com/flybywiresim/a32nx/blob/master/fbw-a32nx/src/systems/instruments/src/ND/elements/WindIndicator.tsx is correct.